values.yaml

```diff
@@ -20,9 +23,10 @@ injector:
   externalVaultAddr: ""
 
   # image sets the repo and tag of the vault-k8s image to use for the injector.
+  # image points to openlabred/vault-k8s repository, pending https://github.com/hashicorp/vault-k8s/pull/126
   image:
-    repository: "hashicorp/vault-k8s"
-    tag: "0.3.0"
+    repository: "openlabred/vault-k8s"
```

Comment:
This needs to be `hashicorp`

Reply:
I guess will be tag: 0.4.0 from hashicorp right?

Reply:
We'll update this on our end for the next release.